src/source/Main.scala

```diff
@@ -282,6 +285,15 @@ object Main {
     } else {
       None
     }
+    val objcSwiftBridgingHeaderWriter = if (objcSwiftBridgingHeaderOut.isDefined) {
```

Comment:
Sorry, I miss read your statement. 

Now instead of asking for the bridging header path `--objc-swift-bridging-header-out "$temp_out/objc/TextSort-Bridging-Header.h"` it only require the bridging header name `--objc-swift-bridging-header "TextSort-Bridging-Header"` and the file will be generated to the objc output directory.

Please let me know if this is ok to you, and if there are any more changes required to get this merged.

Thanks